.coafile

```diff
@@ -0,0 +1,14 @@
+[all]  
+bears = JSONFormatBear
+indent_size = 2
+files = **.json 
+
+[whitespace]
+bears = SpaceConsistencyBear
+use_spaces = True 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `whitespace`.*

The issue can be fixed by applying the following patch:

```diff
--- a/.coafile
+++ b/.coafile
@@ -5,7 +5,7 @@
 
 [whitespace]
 bears = SpaceConsistencyBear
-use_spaces = True 
+use_spaces = True
 files = **.json, .coafile
 
 [length]
```